kotlinx-coroutines-core/common/src/CoroutineStart.kt

```diff
@@ -71,7 +71,7 @@ public enum class CoroutineStart {
      *
      * **Note: This is an experimental api.** Execution semantics of coroutines may change in the future when this mode is used.
      */
-    @ExperimentalCoroutinesApi
+    @ExperimentalCoroutinesApi  // Since 1.0.0, no ETA on stability
```

Comment:
Unlike ATOMIC, this mode does not seem problematic to stabilize. It has pretty well-defined behavior. 

However, it was originally envisioned as a performance optimization for UI thread callbacks which wound start coroutines in `UNDISPATCHED` mode to avoid extra dispatch, but this use-case is exceedingly rare nowadays since we have a stable `immediate` dispatcher which gives the same kind of optimization, so it does no seem necessary anymore to hurry up with stabilization of this mode.